script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
 -- Page 791, Offset 13003.
 -- よしんば話が漏れても噂止まりだ。事故現場に死体があるのは不思議じゃねえんだしな」
-Even if it leaks, it'll be nothing more than a rumor. Besides, there's nothing incredible about dead bodies being present at the scene of an accident."
+'Sides, even if anyone does let it slip, it'll just be a rumor.
+ Not like there's anything unbelievable about there being bodies at the scene of an accident anyhow."
 }
 [sha:48e47085a1a91a3e70c500924664bbc64955c91a]{
 -- Page 792, Offset 13004.
 -- 「そりゃ不思議じゃねぇですけど、不自然ではありますよぉ？　どこにこんだけの死体があったんだって話ですし」
-"Well, it might not be %{i}incredible%{/i}, but isn't there something unnatural about it? Like, where'd all these bodies come from?"
+"Well, sure, it might not be %{i}unbelievable%{/i} but it's still a little %{i}unusual%{/i}, right?
+ Like, where'd all these bodies come from?"
 }
 [sha:5e902f69c2a556c008e3495fc471a5739157d5ae]{
 -- Page 793, Offset 13005.
 -- 「そりゃあオマエ、ここで発見された死体は今夜ここにいた哀れな一般人だろうが。発見された死体イコール、さっきまで生きていた人間ってのが自然だろ？」
-"Listen you, the corpses we found here are the bodies of the poor civilians who were here tonight. Dead bodies found equal the number of previously living people, capeesh?"
+"Listen dipshit, any corpses we turn up here are just the bodies of the poor unfortunate civilians who happened to be here tonight.
+ One dead body equals one human that was definitely alive earlier today, capisce?"
 }
 [sha:9b2c6ba8f13512c5a72dc0dbea571fdea981b2d2]{
 -- Page 793, Offset 13006.
 -- 「げ。マジですか。そんな方便で通るんスか！」
-"Geh. Seriously? Will something so convenient work?"
+"%{i}Geh%{/i}. Seriously? Will something that convenient work?"
 }
 [sha:712c98a29b2debbd637b7ef7d84c82367de45e7d]{
 -- Page 794, Offset 13007.
 -- 「そこを通すんだよ。いやあ、公園だったのが幸いしたぜ。無関係な人間が山ほどいたコトの理由になる」
-"It will. Ahh, we're lucky it happened in a public park. We can use the fact that there were lots of unrelated people present."
+"Oh it'll work.
+ %{i}Hooo%{/i}, man we lucked out that this went down in a public park.
+ Perfect excuse for why there were a bunch a unrelated people in the same place."
 }
 [sha:260e6fea6049d4e3def888f6142f9558abb09066]{
 -- Page 795, Offset 13008.
 -- 「理由さえあればあとは力仕事だわな。何の為にうちのジジイが権力バラまいてると思ってやがる。押さえるのは<上|トップ>だけでいいんだよ。下っ端や真ん中が不審がっても関係ねえ。最後にまとめる人間が握りつぶせばそれでおしまいだ」
-"As long as we have a pretext, the rest is just a matter of manpower. Why do you think my old man's handing out power for? The only people we have to keep in check are the ones on top. It doesn't matter if the underlings or those in the middle get suspicious. As long as the people in charge don't let anything through, it'll go nowhere."
+// Reading 散播く as _wielding_ political power rather than _disseminate_ (i.e, 散播き政治)
+// since that seems to make more sense in ctx
+"So long as we have a cover story, the rest is just a matter of legwork.
+ Why d'ya think my old man's been leaning on people left and right?
+ All you gotta do is put pressure on the guys at the top.
+ Doesn't make a fat bit of difference if the small fry or middle managers get suspicious.
```

Comment:
if a small fry or a middle manager gets suspicious
if small fries or middle managers get suspicious

Reply:
Hmm, not sure about this one. I feel like it should be plural here (doesn't matter even if a bunch of lesser folk are suspicious) but 'small fries' doesn't read right to me as the plural form of small fry. I think it's just a defective noun.

Reply:
Seems like [both are correct](https://www.merriam-webster.com/dictionary/small-fry) but I really, really don't care in this case